modules/oe_theme_helper/templates/oe-theme-helper-site-navigation.html.twig

```diff
@@ -10,8 +10,10 @@
  * @ingroup themeable
  */
 #}
-{{ pattern('navigation_menu', {
-  label: 'Menu'|t,
-  items: menu_items,
-  site_name: site_name,
-}, 'vertical') }}
+{% if menu_items %}
```

Comment:
We don't need to check the menu_items twice, the pattern already checks if there is content to be rendered.